hq/markdown/wazuh.md

```diff
@@ -1,10 +1,9 @@
 # Wazuh
-
+![Wazuh logo](https://www.axonius.com/hubfs/Adapter%20Logos/wazuh_adapter.png)
```

Comment:
could we pul the logo in source control rather than pulling it in from a third party here?